swatch-product-configuration/src/main/resources/subscription_configs/Ansible/Ansible_Aap_Managed.yaml

```diff
@@ -0,0 +1,22 @@
+---
+platform: Ansible
+
+id: ansible-aap-managed
+
+variants:
+  - tag: ansible-aap-managed
+    productNames:
+      - Ansible Automation Platform Managed App
+
+defaults:
+  variant: ansible-aap-managed
+  sla: PREMIUM
+  usage: PRODUCTION
+
+serviceType: Ansible Managed Node
+
+contractEnabled: true
+
+metrics:
+  - id: Managed-host
+    awsDimension: managed_host
```

Comment:
```suggestion
  - id: Managed-hosts
    awsDimension: managed_hosts
```

Reply:
Done

Reply:
changed to Managed-nodes instead of Managed-hosts